indico/modules/events/controllers/preparation.py

```diff
@@ -0,0 +1,42 @@
+# This file is part of Indico.
+# Copyright (C) 2002 - 2020 CERN
+#
+# Indico is free software; you can redistribute it and/or
+# modify it under the terms of the MIT License; see the
+# LICENSE file for more details.
+
+from __future__ import unicode_literals
+
+from uuid import uuid4
+
+from flask import jsonify, request
+from webargs import fields
+
+from indico.legacy.common.cache import GenericCache
+from indico.web.args import use_kwargs
+from indico.web.rh import RH
+
+
+class RHPrepareEvent(RH):
+    """Prepare a new event, store it using GenericCache util, and create a UUID."""
+
+    CSRF_ENABLED = False
+    _cache = GenericCache('event-preparation')
+
+    @use_kwargs({
+        'title': fields.String(missing=""),
+        'start_dt': fields.String(missing=""),
```

Comment:
We can use the custom `NaiveDateTime` field type (from `indico.util.marshmallow`) here to get an actual datetime object instead of just a string.

Reply:
Updated.